src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles 
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties 
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const propertiesArr = sourceString.replace(/[\r\n]/gm, '').split(';');
+  const propObj = propertiesArr.reduce((acc, item) => {
+    const splitIndex = item.indexOf(':');
+    const key = item.slice(0, splitIndex).trim();
+    const value = item.slice(splitIndex + 1).trim();
```

Comment:
Maybe just something like this: 

```suggestion
const [key, value] = cssProperty.split(':');
```